### PR TITLE
Support specifying an ssl client certificates key passphrase

### PR DESCRIPTION
**Pull request by Ponyboy47**
I'm in need of this functionality and #596 is waiting on a PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -921,6 +921,7 @@ kafka = Kafka.new(
   ssl_ca_cert: File.read('my_ca_cert.pem'),
   ssl_client_cert: File.read('my_client_cert.pem'),
   ssl_client_cert_key: File.read('my_client_cert_key.pem'),
+  ssl_client_cert_key_password: 'my_client_cert_key_passphrase',
```
Comment (mensfeld) [inline README.md]: I see an inconsistency here. Why is it called `password` when the value and PR message says `passphrase`?

File: lib/kafka/client.rb [hunk 2]
```diff
@@ -46,6 +46,9 @@ class Client
     # @param ssl_client_cert_key [String, nil] a PEM encoded client cert key to use with an
     #   SSL connection. Must be used in combination with ssl_client_cert.
     #
+    # @param ssl_client_cert_key_password [String, nil] the passphrase required to read the
```
Comment (mensfeld) [inline lib/kafka/client.rb]: missmatch betwen password and passphrase.

**Comment by Ponyboy47** [comment]
No, I should be able to wait for the next release. We're just using our own fork for now

**Comment by mensfeld** [comment]
@nguyenquangminh0711 do we merge it?

**Comment by nguyenquangminh0711** [comment]
@mensfeld Of course, I approved so that the author can merge it 😄 But I'll merge it now

**Pull request merged**